.rubocop.yml

```diff
@@ -0,0 +1,213 @@
+AllCops:
```

Comment:
I don't see rubocop declared as a dependency, did you install it locally with `gem install`?
I think it's better to have it as a dependency in the `Gemfile` so that the project brings everything it needs (and in the expected version)  - `gem 'rubocop', require: false`.

It'd also be good to integrate the rake (Ruby's task runner) task:
https://rubocop.readthedocs.io/en/latest/integration_with_other_tools/

Finally, looks like these settings create some offenses around double quotes.
To start with a clean state, either change the rule or fix the offenses (rubocop can do it for you).